tests/testthat/test-input-time-series.R

```diff
@@ -0,0 +1,121 @@
+test_that("data can be formatted for ART input time series", {
+  data <- prepare_input_time_series_art(a_hintr_data$art_number,
+                                        a_hintr_data$shape)
+
+  expect_true(nrow(data) > 100) ## Check that we have read out some data
+  expect_setequal(colnames(data),
+                  c("area_id", "area_name", "area_level_label","area_level",
+                    "time_step", "time_period", "plot", "value"))
+})
+
+test_that("data can be formatted for ANC input time series", {
+  data <- prepare_input_time_series_anc(a_hintr_data$anc_testing,
+                                        a_hintr_data$shape)
+  expect_true(nrow(data) > 100) ## Check that we have read out some data
+  expect_setequal(colnames(data),
+                  c("area_id", "area_name", "area_level", "area_level_label",
+                    "age_group", "time_period", "time_step", "plot", "value"))
+})
+
+test_that("ART data can be aggregated by time and space", {
+
+  art_number <- readr::read_csv(a_hintr_data$art_number, show_col_types = FALSE)
+
+  # Make dummy quarterly data
+  art_q <-art_number %>%
+    dplyr::mutate(art_current = art_current/4,
+           calendar_quarter = paste0("CY",year, "Q1")) %>%
+    rbind(art_number %>%
+            dplyr::mutate(art_current = art_current/4,
+                   calendar_quarter = paste0("CY",year, "Q2"))) %>%
+    rbind(art_number %>%
+            dplyr::mutate(art_current = art_current/4,
+                   calendar_quarter = paste0("CY",year, "Q3"))) %>%
+    rbind(art_number %>%
+            dplyr::mutate(art_current = art_current/4)) %>%
+    dplyr::select(-c(art_new))
+
+  dir <- tempdir()
+  art_q_file <- paste0(dir, "art_q.csv")
+  readr::write_csv(art_q, art_q_file)
+
+  data <- prepare_input_time_series_art(art_q_file,
+                                        a_hintr_data$shape)
+
+  ## Check that data has been aggregated to all levels in hierarchy from baseline
+  ## determined by highest area_level in program data
+  shape <- sf::read_sf(a_hintr_data$shape)
+  art_number <- art_number %>%
+    dplyr::left_join(shape %>% dplyr::select(area_id, area_level), by = "area_id")
+
+  # Check data has been aggregated from correct baseline
+  expect_equal(unique(art_number$area_level), max(data$area_level))
+  # Check data has been aggregated from baseline to lowest level in hierarchy
+  shape_level <- unique(shape$area_level)[unique(shape$area_level) <= unique(art_number$area_level)]
+  expect_equal(shape_level, unique(data$area_level))
```

Comment:
Sorry this is a bit picky but can you switch these around?
```suggestion
  expect_equal(unique(data$area_level), shape_level)
```
convention is to have the thing you are testing on the LHS.